### ISSUE TITLE
Investigate new PathPlanner changes

### ISSUE THREAD

**Issue by CrispyBacon1999** [Hemlock5712/2024-Robot#12]
Pathplanner had some pretty major changes in how things are done, including in the coordinate system. We need to relearn this and make sure we know exactly how it works.

**Comment by JosephTLockwood** [comment]
I believe that we will need to switch

```
Alliance alliance = allianceOptional.get();
if (alliance == Alliance.Blue) {
  poseEstimation = results.getBotPose3d_wpiBlue();
} else {
  poseEstimation = results.getBotPose3d_wpiRed();
}
```

to

```
poseEstimation = results.getBotPose3d_wpiBlue();
```
https://www.chiefdelphi.com/t/pathplanner-2024-beta/442364/370?u=joeojazz

**Comment by JosephTLockwood** [comment]
Will be closed on pull from the template

**Comment by CrispyBacon1999** [comment]
Fixed with #29
